### ISSUE TITLE
Is there a REST api?

### ISSUE THREAD

**Issue by Fexxix** [filecxx/FileCentipede#828]
I did notice that you can set up remote access in the settings but can that be used for making calls to the a rest api or does it not exist?

**Comment by filecxx** [comment]
No, but it can involve data swapping in JSON format. Unfortunately, the API is currently undocumented.

**Comment by Fexxix** [comment]
> No, but it can involve data swapping in JSON format. Unfortunately, the API is currently undocumented.
I see, please let me know when its documented.
